Aptoide/aptoide-client-v8
# [ASV-1377] MoPub Native Ads in Prod

**What does this PR do?**

   This PR aims at adding native ads in production. These ads can be seen in Home (highlighted bundle), Search (only on all stores tab) and AppView on both similar app bundles (before and after starting the download).

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] HomeFragment.java

**How should this be manually tested?**

Open the app and check if you see ads on all of the aforementioned sections. Notice that in order to see ads, you will probably need a VPN (ask me if you need help) and there is ab testing setup (90-10).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1377](https://aptoide.atlassian.net/browse/ASV-1377)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
